src/index.ts

```diff
@@ -67,10 +69,12 @@ const listrTasks = [
 
   // Linting
   eslint,
+  stylelint,
   prettier,
 
   // Version and configuration checks
   eslintConfigIsValid,
+  stylelintConfigIsValid,
   preflightIsLatestVersion,
```

Comment:
very similar to https://github.com/upleveled/preflight/pull/377/files#r1172876748

should we have the stylelint check after eslint or doing this alphabetically